src/Media/Media.js

```diff
@@ -0,0 +1,12 @@
+import React, {Component} from 'react';
+
+export default class Media extends Component {
```

Comment:
I think for consistency, we'd want to re-export e.g. `Media.Body`, `Media.Left`, &c. from here, and use those by preference in examples. @jquense WDYT?